bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/gateway.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fineoffsetweatherstation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="gateway">
+		<label>Gateway Device</label>
+		<category>NetworkAppliance</category>
+		<config-description-ref uri="thing-type:fineoffsetweatherstation:gateway"/>
+	</bridge-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wind-speed">
+		<item-type>Number:Speed</item-type>
+		<label>Wind Speed</label>
+		<category>Wind</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
```

Comment:
Please remove these tags.

Reply:
Why should these tags be removed? That is exactly what is being measured, isn't it?

Reply:
What exactly is being measured? A quantity in which unit?

Reply:
the wind speed in m/s

Reply:
So wind speed is being measured. I'm sure if "Measurement" + "Wind" makes sense. @lolodomo - can you help?

Reply:
System channel `system.wind-speed` also uses this combination of tags, so resolving comment.